homeassistant/components/plex/media_player.py

```diff
@@ -319,23 +317,7 @@ def __init__(self, config, device, session, plex_sessions,
 
         self.refresh(device, session)
 
-        # Assign custom entity ID if desired
-        if self.config.get(CONF_USE_CUSTOM_ENTITY_IDS):
-            prefix = ''
-            # allow for namespace prefixing when using custom entity names
-            if config.get("entity_namespace"):
-                prefix = config.get("entity_namespace") + '_'
-
-            # rename the entity id
-            if self.machine_identifier:
-                self.entity_id = "%s.%s%s" % (
-                    'media_player', prefix,
-                    self.machine_identifier.lower().replace('-', '_'))
-            else:
-                if self.name:
-                    self.entity_id = "%s.%s%s" % (
-                        'media_player', prefix,
-                        self.name.lower().replace('-', '_'))
+        self.entity_id = "{}.{}_{}".format('media_player', 'plex', self.name)
```

Comment:
Don't set any entity_id. That will not break the system because the entity component has register that already and new device follow the normal process.

Reply:
It would be useful for identifying the newly created entities. Instead of `media_player.chrome` if a web client connects, it would show `media_player.plex_chrome` which has more meaning.

Instead, would setting `name` in this pattern be preferred?